packages/utils/src/feature-flags/workflows.ts

```diff
@@ -3,6 +3,6 @@ import { FeatureFlagTypes } from "@medusajs/types"
 export const WorkflowsFeatureFlag: FeatureFlagTypes.FlagSettings = {
   key: "workflows",
   default_val: false,
-  env_key: "MEDUSA_FF_WORKFLOWS",
+  env_key: "MEDUSA_FF_WORKFLOW_ENGINES",
```

Comment:
question: Is this not a separate FF? 🤔

Reply:
Yes, my find and replace went too far haha... thanks